mllib/src/main/scala/org/apache/spark/ml/feature/PCA.scala

```diff
@@ -53,7 +53,8 @@ private[feature] trait PCAParams extends Params with HasInputCol with HasOutputC
 
 /**
  * :: Experimental ::
- * PCA trains a model to project vectors to a low-dimensional space using PCA.
+ * PCA trains a model to project vectors to a lower dimensional space of the top [[PCA!.k]]
```

Comment:
while you're looking @holdenk , is this right with the "!"?  I saw this somewhere else and without that it seems like it can't find `k`, but I couldn't find anything about it in scaladoc.